server/src/main/java/org/opensearch/cluster/node/DiscoveryNode.java

```diff
@@ -61,7 +61,7 @@
 import java.util.stream.Stream;
 
 import static org.opensearch.node.NodeRoleSettings.NODE_ROLES_SETTING;
-import static org.opensearch.node.remotestore.RemoteStoreNodeAttribute.REMOTE_STORE_NODE_ATTRIBUTE_KEY_PREFIX;
+import static org.opensearch.node.remotestore.RemoteStoreNodeAttribute.*;
```

Comment:
It would be good to avoid wildcard imports. If you are using IntelliJ, this is how you would do it: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

The spotless checks would also fail because of this.

Reply:
Removed wildcard imports.